src/Controller/Component/CrudComponent.php

```diff
@@ -218,9 +218,7 @@ public function execute($controllerAction = null, $args = [])
         $this->_action = $controllerAction ?: $this->_action;
 
         $action = $this->_action;
-        if (empty($args)) {
-            $args = $this->_request->params['pass'];
-        }
+        $args = (!empty($args)) ? [$args] : $this->_request->params['pass'];
```

Comment:
This `[$args]` is something i wasn't sure about.
When i call `$this->Crud->execute(null, ['key1', 'key2'])` without the `[$args]`, the `BaseAction->handle()` do `return call_user_func_array([$this, $method], $args)`, then only my "key1" will go to the `_findMethod()`.

In this case, or i use that `$args = [$args]`, or user will have to use  

    $this->Crud->execute(null, [["key1","key2"]]);